beluga_example/bags/perfect_odometry/metadata.yaml

```diff
@@ -1,36 +1,42 @@
 rosbag2_bagfile_information:
   version: 5
-  storage_identifier: "sqlite3"
+  storage_identifier: ""
   duration:
-    nanoseconds: 129982460028
+    nanoseconds: 138159689628
   starting_time:
-    nanoseconds_since_epoch: 1676668727259050014
-  message_count: 4595
+    nanoseconds_since_epoch: 1676668727219043066
+  message_count: 24441
   topics_with_message_count:
+    - topic_metadata:
+        name: /clock
```

Comment:
@nahueespinosa meta: it's OK to have a `/clock` topic, but it shouldn't be necessary, `rosbag2` can be instructed to publish it.